Web/jqueryassignment2/qn2/WebContent/Assignment2.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() 
+{
+	$("#empName").change(function() 
+	{
+		loadInfo();
+	});
+});
+function loadInfo()
```

Comment:
I think it is jQuery assignment therefore you should have used jQuery's Ajax APIs directly. Don't you?